modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,201 @@
+= Recovery Mode
+:description: 
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and restore the failed cluster. Topic reads and writes are disabled.
+
+== About recovery mode
+
+Recovery mode starts a cluster with limited functionality and deactivates normal traffic and data operations for a cluster. Redpanda does not load user-managed partitions on disk. This provides a more stable environment for troubleshooting issues and restoring the cluster to a usable state. 
+
+Recovery mode helps to prevent triggering a storm of partition leader elections and replication (such as increased learner replica recovery load after a long broker downtime) that may occur on startup. 
+
+Normal traffic and especially malicious traffic can disrupt availability for all users, including admin users. Recovery mode helps avoid placing the cluster under a heavy load of Kafka API and HTTP Proxy connections as you work to resuscitate the cluster.
+
+== Requirements
+
+To enable recovery mode, ensure that you have [rpk] installed, and you have local access to machines running Redpanda.
+
+Brokers can only enter recovery mode as it starts up, and not while it is already running. First set the broker configuration property to enable recovery mode, and then do a broker restart.
+
+== Use recovery mode
+
+To use recovery mode, first configure the broker to enable recovery mode when a broker starts.
+
+. Run the `rpk redpanda mode` command to set the `recovery_mode_enabled` broker configuration property to `true`.
+// TODO: confirm whether rpk reference is missing this update
++
+[,bash] 
+---- 
+rpk redpanda mode recovery
+----
+
+. Start the cluster. You can run `rpk cluster health` to check whether the cluster has entered recovery mode.
```

Comment:
We should show the output of the cluster health command here @kbatuigas

Reply:
Good point, it looks like this:
```
$ rpk cluster health
CLUSTER HEALTH OVERVIEW
=======================
Healthy:                          true
Unhealthy reasons:                []
Controller ID:                    0
All nodes:                        [0 1 2]
Nodes down:                       []
Nodes in recovery mode:           [0 1 2]
Leaderless partitions (0):        []
Under-replicated partitions (0):  []
```
(note the "Nodes in recovery mode:" field)